packages/documentation-site/patternfly-docs/content/developer-resources/dark-theme-handbook.md

```diff
@@ -6,82 +6,26 @@ section: developer-resources
 
 ## Enabling dark theme
 
-Dark theme styles ship with PatternFly CSS by default. To enable dark theme, add the class `pf-v6-theme-dark` to your application's `<html>` tag. This class can be added dynamically to toggle dark theme on and off in your application.
+Dark theme styles ship with PatternFly CSS by default. To enable dark theme, add the class `pf-version-theme-dark` (for example, `pf-v6-theme-dark`) to your application's `<html>` tag. This class can be added dynamically to toggle dark theme on and off in your application.
```

Comment:
In the core docs, if we want to refer to something like "version" in a string like that, we will write it like `pf-[version]-theme-dark`, just so it's better understood you shouldn't use `pf-version-theme-dark`

Reply:
oo this is really helpful to know -- that looks better